src/utils/transitions.js

```diff
@@ -228,6 +242,25 @@ export function delayedFloatUpFn({ element, onComplete }) {
 	}, springDelay);
 }
 
+export function staggeredFloatUpFn({ element, onComplete }) {
+	const elementStyler = styler(element);
+	const styleFn = toStaggeredFloatyY;
+	const animationDirection = animateUp;
+	const distanceBase = 20;
+	const distanceMultiplier = 5;
+	const endValue = distanceBase + (element.dataset.loadIndex * distanceMultiplier);
```

Comment:
oh interesting, so `data-load-index` has to be added to each DOM element because that tells the JS how much to animate it by